src/test/java/edu/hm/hafner/analysis/parser/ClangAnalyzerPlistParserTest.java

```diff
@@ -0,0 +1,66 @@
+package edu.hm.hafner.analysis.parser;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import static edu.hm.hafner.analysis.assertj.Assertions.*;
+
+import edu.hm.hafner.analysis.assertj.SoftAssertions;
+
+
+/**
+ * Tests for {@link TaglistParser}.
+ */
+class ClangAnalyzerPlistParserTest extends AbstractParserTest {
+    ClangAnalyzerPlistParserTest() {
+        super("clang-analyzer-test.txt");
+    }
+
+    @Override
+    protected ClangAnalyzerPlistParser createParser() {
+        return new ClangAnalyzerPlistParser();
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        softly.assertThat(report).hasSize(3);
+
+        softly.assertThat(report.get(0))
+                .hasSeverity(Severity.WARNING_NORMAL)
+                .hasLineStart(538)
+                .hasLineEnd(538)
```

Comment:
It does not harm but typically you do not need to test the `*end` methods since you are not touching them

Reply:
Done